<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccountWindow</class>
 <widget class="QDialog" name="CreateAccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_name">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_username"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_password"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_email"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_phone_number"/>
   </item>
   <item>
    <widget class="QPushButton" name="create_button">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exitBtn">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
